.github/workflows/ubuntu-build.yml

```diff
@@ -46,6 +48,11 @@ jobs:
         run: |
           cmake --build build -j$(nproc)
 
+      - name: CTest Run
+        shell: bash
+        run: |
+          ctest --test-dir ./build --output-on-failure
```

Comment:
Could you please do something like this here?

```
echo 'DISABLED ctest --test-dir ./build --output-on-failure'
```

Just enough that GHA succeed. I think it's best to merge and do follow-on work separately.

(Aiming for merging on Monday.)